.pipelines/pipeline.yaml

```diff
@@ -322,24 +322,38 @@ stages:
       displayName: Cilium
       pipelineBuildImage: "$(BUILD_IMAGE)"
       testDropgz: ""
+      clusterType: swift-no-kube-proxy-up
```

Comment:
can clusterType be `cilium-podsubet-up`

Reply:
This parameter `clusterType` is calling the respective commands from [hack/aks/makefile](https://github.com/Azure/azure-container-networking/blob/cc251ca58a0fe6009a7c80c167ca52887169dce3/hack/aks/Makefile#L156). We can change them to whatever we want.

Reply:
let's change the clustertypes to match the new hackfile commands